tests/test_contexts.py

```diff
@@ -51,6 +51,13 @@ def test_init():
     assert c.bools == [(True, False), (False, True)]
 
 
+def test_shape():
+    c = Context(('spam', 'eggs'), ('camelot', 'launcelot'),
```

Comment:
Can we use the `context` fixture (example context)?

Current example is ambiguous.

Reply:
Aggree, I will add this :)

Reply:
It's probably even better to make this as simple doctest, cf e.g.:
https://github.com/xflr6/concepts/blob/5e785a0ada07ad06a1bcee5cd3fcf555d320f107/concepts/contexts.py#L541-L550